gym_multigrid/core/agent.py

```diff
@@ -1,60 +1,75 @@
 import enum
 import math
 from typing import Type, TypeVar
-
 import numpy as np
 from numpy.typing import NDArray
```

Comment:
There should be a blank line here to separate the 3rd party libs and our modules